stake-pool/program/src/processor.rs

```diff
@@ -1890,6 +1930,7 @@ impl Processor {
         let clock = &Clock::from_account_info(clock_info)?;
         let token_program_info = next_account_info(account_info_iter)?;
         let stake_program_info = next_account_info(account_info_iter)?;
+        let manager_fee_info = next_account_info(account_info_iter)?;
```

Comment:
Don't forget to check that this is the right manager fee account! Otherwise, I can just put my own account in twice

Reply:
hmm yeah, it also reminds me, we need to include the manager field in the withdraw command in the cli